src/hooks/useSSO.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from "react";
+import firebase from "firebase/app";
+
+import { SSO_PROVIDER_ID } from "secrets";
+
+export const useSSO = () => {
+  const hasSSOProvider = SSO_PROVIDER_ID !== undefined;
+
+  const signInSSO = useCallback(() => {
```

Comment:
[NB]

Maybe something like this? Naming flows weirdly currently:
```suggestion
  const signInWithSSO = useCallback(() => {
```

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1546/commits/62de340d0e1eda6f11434e9fa870896e6698c0f7